plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -192,6 +192,17 @@ proc mayaUsdTranslatorExport_SetConvertMaterialsToCheckboxes(string $arg, int $e
     }
 }
 
+proc mayaUsdTranslatorExport_DisableConvertMaterialsToCheckboxes() {
+    // If we are using a legacy material export mode, then disable the checkboxes
+    // since this material export mode is not compatible with multi-material export.
+    string $conversions[] = `mayaUSDListShadingModes -export -useRegistryOnly`;
+    for ($conversion in $conversions) {
+        string $opt = `mayaUSDListShadingModes -eo $conversion -useRegistryOnly`;
+        string $widget = $opt + "_ConvertMaterialsToCheckBox";
+        checkBoxGrp -e -v1 false -en false $widget;

```

Comment:
It is possible to get into a state where the string $widget will not be a valid checkBoxGrp? Should we add a if `checkbBoxGrp -exists $widget)`